### DISCUSSION TITLE
Welcome to sliderule-web-client Discussions!

### DISCUSSION THREAD

**Discussion by cugarteblair** [SlideRuleEarth/sliderule-web-client#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mataleogml** [comment]
Hey this is Gabriel! I am one of the two student designer who perviously worked on designing the web interface for SlideRule!

**Reply by cugarteblair** [reply]
What do you think of it now?
